examples/jupyter_notebooks/beta_development/sonde_tropopause_obs.ipynb

```diff
@@ -0,0 +1,267 @@
+{
```

Comment:
Let's take this out for version 1.1 release and keep it on your fork. I made an issue to work on adding this in at a future time too so that someone else can work off of your code.